pkg/k8s/apps/translate.go

```diff
@@ -56,12 +57,15 @@ func (tr *Translation) translate() error {
 	delete(tr.App.ObjectMeta().Annotations, model.StateBeforeSleepingAnnontation)
 
 	tr.DevApp = tr.App.DevClone()
+	annotationsString, _ := json.Marshal(tr.App.ObjectMeta().Annotations)
 
 	tr.App.ObjectMeta().Annotations[model.AppReplicasAnnotation] = strconv.Itoa(int(replicas))
+	tr.App.ObjectMeta().Annotations[model.AppAnnotationsAnnotation] = string(annotationsString)
```

Comment:
For now, let's not store them. On `okteto down` we just delete the ones in `tr.Dev.Metadata.Annotations`

Reply:
What if you modify one environment from the original deployment? This would cause a variable to be deleted from the original deployment which may result in the application being left in an undesired status

Reply:
Do you have a use case in mind? I think is very unlikely, but if you want to handle it, I would add the previous annotation state just if needed

Reply:
I was thinking about connecting to a different database from the one in staging, for example I could connect to my local db instead of using a shared db

Reply:
You would use envvars for that, no?

Reply:
Yes, you're right, I was mixing up both. I won't store them for now